### ISSUE TITLE
Trouble with thumbnails 

### ISSUE THREAD

**Issue by kjrawson** [CollegeOfTheHolyCross/dta_sufia#70]
Hi, Steven.
I've been working with a collection that is now posted––Riki Anne Wilchins' Flyers: https://www.digitaltransgenderarchive.net/catalog?f%5Bcollection_name_ssim%5D%5B%5D=Riki+Anne+Wilchins%27+Flyers. 

But, none of the thumbnails are showing up. The files are there correctly if you click through, so I am not sure where the breakdown is happening.

Thanks!

**Comment by kjrawson** [comment]
Also, you can see in the collection description that the student added a link. Is it possible for that to be hyperlinked? Thank you!
https://www.digitaltransgenderarchive.net/collections/bv73c042m

**Comment by scande3** [comment]
I restarted the background process with then generated the thumbnails for the first issue.

Added the link to that collection as well. Fixed two views so it would show up as a link in them via commit a0ad22b1aa1ed1876e3e5332d02824dc3a7a518d. Will handle this differently in the near future so that just entering a text link is all that is required and the code makes anything with "http://" a link automatically on the page.
